Src/AutoFixture/Kernel/OmitRecursionGuard.cs

```diff
@@ -1,34 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Collections;
-
-namespace Ploeh.AutoFixture.Kernel
-{
-    public class OmitRecursionGuard : RecursionGuard
```

Comment:
Since the file is not actually being compiled, and it looks like it was added to the Git repository by accident, I think it ought to be removed.

This should happen in a separate pull request, because it ought to be removed from *master* as well as *v4*, as far as I can tell.

If you want the 'credit', I'll be happy to let you do this, but I can also do it myself if you don't want to bother with this.